eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -173,6 +175,20 @@ jobs:
         customPrepArgs="${customPrepArgs} --no-sdk --no-artifacts"
         prepSdk=false
       fi
+
+      if [[ '${{ parameters.runOnline }}' == 'False' ]]; then
```

Comment:
Yes, but that would be "cheating". The intent is to exercise the workflow that would be used by partners/customers making use of the prep script. To do that, we'd pass in the feed/credentials directly to the prep script and have it do the work. And since the feed can be anything, we can't have the prep script call the SetupNugetSources script.

Reply:
Makes sense, I overlooked the end user scenario and was thinking only about how to optimize the build definition.